assignment-client/src/AssignmentClientMonitor.cpp

```diff
@@ -201,63 +202,53 @@ void AssignmentClientMonitor::checkSpares() {
     }
 }
 
+void AssignmentClientMonitor::handleChildStatusPacket(QSharedPointer<NLPacket> packet) {
+    QUuid senderID = QUuid::fromRfc4122(packet->read(NUM_BYTES_RFC4122_UUID));
```

Comment:
I guess we could make this more efficient - construct a `QByteArray::fromRawData` and just seek on the `packet` to move forwards